base_classes/NXbeam.nxdl.xml

```diff
@@ -142,4 +142,72 @@
             for a summary of the discussion.
         </doc>
     </attribute>
+
+    <field name="depends_on" minOccurs="0" type="NX_CHAR">
+        <doc>
+            NeXus path to the beam direction (and location) at beamline component or sample
+        </doc>
```

Comment:
Proposed revision, workshopped in the code camp.  Instead of "NeXus path to the beam direction (and location) at beamline component or sample", we could use the following:

The NeXus coordinate system defines the Z axis to be along the nominal beam direction. This is the same as the McStas coordinate system (see :ref:`Design-CoordinateSystem`). However, the additional transformations needed to represent an altered beam direction can be provided using this depends_on field that contains the path to a NXtransformations group. This could represent redirection of the beam, or a refined beam direction.

Reply:
Waiting for reply from @PeterC-DLS

Reply:
Adding a transformation matrix back to McStats (somewhere in `NXentry`) is fine by me and a reminder note here.